cdap-proto/src/main/java/co/cask/cdap/proto/artifact/AppRequest.java

```diff
@@ -41,19 +41,19 @@ public AppRequest(ArtifactSummary artifact) {
   }
 
   public AppRequest(ArtifactSummary artifact, @Nullable T config) {
-    this(artifact, config, null, null, true);
+    this(artifact, config, null, null, null);
```

Comment:
Don't like the idea of having a boolean as null we can initialize with true or false whatever the default value is here.